arrays/arrays.c

```diff
@@ -7,7 +7,7 @@ typedef struct Array
 {
     int capacity;    // How many elements can this array hold?
     int count;       // How many states does the array currently hold?
-    char **elements; // The string elements contained in the array (it's an array of strings!)
+    char **elements; // The string elements contained in the array (it's an array of string pointers!)
```

Comment:
yeah! they just happen to be so similar in C, but it's good to separate the concepts